aea/cli/run.py

```diff
@@ -95,7 +95,7 @@ def _verify_or_create_private_keys(ctx: Context) -> None:
     else:
         try:
             _try_validate_fet_private_key_path(fetchai_private_key_path)
-        except FileNotFoundError:
+        except FileNotFoundError:  # pragma: no cover
```

Comment:
Again I think you can mock the function or since you are passing the path now as an str, pass a wrong type for example `feetchai_private_key_path.pem` rather than `.txt`